SECURITY.md

```diff
@@ -0,0 +1,64 @@
+# Security Policy
+
+## Supported Versions
+
+LoopBack implements the
```

Comment:
I'm struggling whether we should include this section. I think the existing docs page https://loopback.io/doc/en/contrib/Long-term-support.html already serves the purpose of stating the current status of different LB versions. Since this file is in loopback-next, perhaps we can keep this to LB4 specific only?